### ISSUE TITLE
Unable to open tree view

### ISSUE THREAD

**Issue by wvengen** [atom/tree-view#474]
Since Atom commit https://github.com/atom/atom/commit/4300ba6bb3155eb4340eb9bda42a5ab171bd24c4  the treeview doesn't show up anymore. Trying to open it (command "Tree View: Show") appears to do nothing. Commit https://github.com/atom/atom/commit/57610518f7edbc1f16c9ed8ed08dce457b85f34e still works.
Removing `~/.atom` does not change anything.
Running on Ubuntu Linux 15.04 kernel 3.16.0-36-generic x86_64.
Node v0.10.33, npm v2.5.1.

Startup messages (for good and bad commit):
```
$ atom -n -d -f -w
[7409:0609/132223:ERROR:browser_main_loop.cc(170)] Running without the SUID sandbox! See https://code.google.com/p/chromium/wiki/LinuxSUIDSandboxDevelopment for more information on developing with the sandbox on.
App load time: 230ms
[7409:0609/132224:INFO:CONSOLE(49)] "Window load time: 800ms", source: file:///[...]/share/atom/resources/app.asar/static/index.js (49)
[7409:0609/132225:WARNING:accelerator_util.cc(181)] Invalid accelerator token: 
[7409:0609/132225:WARNING:accelerator_util.cc(181)] Invalid accelerator token: 
[7409:0609/132232:WARNING:accelerator_util.cc(181)] Invalid accelerator token: 
```

**Comment by mnquintana** [comment]
Is your window opening with an unsaved buffer (e.g. `untitled`) and nothing else?

**Comment by wvengen** [comment]
Yes. Thanks for your super-fast response!

**Comment by mnquintana** [comment]
Ah okay, so then this is a duplicate of https://github.com/atom/tree-view/issues/68 – what's happening is that after this update, Atom opened not in a directory, and you currently can't open the tree-view unless you have a directory loaded. To fix this, you can just open a project directory in Atom and it should work normally. Thanks for the update!

**Comment by wvengen** [comment]
Ah, I see! I'm not using projects but always start from a directory - then choosing a file from the tree-view. As a workaround, I can change it by either always opening a specific file, or figuring out how to setup projects in Atom. Thanks!
